### PR TITLE
Add flatmap prov information button

### PR DESCRIPTION
**Pull request by Tehsurfer**
Added a information icon which shows flatmap provenance.

Also switched the `loading` to `flatmapReady` as this was needed to know when we can retrieve flatmap provenance info from the `mapImp` object

image

Below is an old version I am keeping here for storage
image

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -68,13 +68,13 @@ export default {
...
-      let taxon = component.mapImp.describes;
-      let id = component.mapImp.addMarker("UBERON:0000948");
+      // console.log(component);
+      // let taxon = component.mapImp.describes;
+      // let id = component.mapImp.addMarker("UBERON:0000948");
       window.flatmapImp =  component.mapImp;
       component.enablePanZoomEvents(true);
       //component.showPathwaysDrawer(false);
-      console.log(taxon, id);
+      // console.log(taxon, id);
```
Comment (alan-wu) [inline src/App.vue:77]: Can we keep the addMarker here please? It helps with checking rather the markers are working. Thanks.

**Review by alan-wu — CHANGES_REQUESTED** [review]
Thanks, it looks good and clean but there are few issues with the context cards

1. The banner image does not match the active species, perhaps the image can be removed.
2. I cannot hide the popup once is on display.
3. The "i" on the icon is transparent instead of solid white, it is not a big issue but it does not match with the others.

**Comment by Tehsurfer** [comment]
I've implemented the feedback but am struggling with #3 as the icon is actually a font which is really hard to size. Have any ideas on how to make the background white?

**Review by alan-wu — APPROVED** [review]
Looks good, thanks.

**Pull request merged**